security-oauth2/src/main/java/io/micronaut/security/oauth2/endpoint/token/response/IdTokenLoginHandler.java

```diff
@@ -31,16 +31,13 @@
 import io.micronaut.security.token.jwt.cookie.AccessTokenCookieConfiguration;
 import io.micronaut.security.token.jwt.cookie.CookieLoginHandler;
 import jakarta.inject.Singleton;
-import java.text.ParseException;
-import java.time.Duration;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.text.ParseException;
+import java.time.Duration;
+import java.util.*;
```

Comment:
is the wildcard import intentional?

Reply:
It was the IDE optimising them. Wildcard imports are allowed as per our Checkstyle config.